jepsen/src/jepsen/nemesis/combined.clj

```diff
@@ -360,6 +360,45 @@
                            :fs    #{:strobe-clock}
                            :color "#A0E9E3"}}}))
 
+(defn file-corruption-nemesis
+  "Wraps [[jepsen.nemesis/bitflip]] and [[jepsen.nemesis/truncate-file]] to corrupt files.
```

Comment:
What's up with the `[[x]]`? Usually you'd write these as `x`, I think, or just bare?

Reply:
Tells codox it's a link:

> Markdown docstrings also support wikilink-style relative links, for referencing other vars. Vars in the current namespace will be matched first, and then Codox will try and find a best match out of all the vars its documenting.

\`x\` does style as source code, but not a link.
And bare leaves it as text.

It's also why mentioning vectors can lead to odd formatting, [[not] [a] [link]] turns into [[not a link]](url).

I was  thinking about doing a doc formatting only PR for `jepsen.nemesis*` before 0.2.8.

Jepsen has good docs! Thanks!